src/L.Routing.Mapzen.js

```diff
@@ -10,14 +10,17 @@
   L.Routing.Mapzen = L.Class.extend({
 
 
-    initialize: function(accessToken, transitmode, costingOptions, otherOptions, options) {
+    initialize: function(accessToken, transitmode, costingOptions, directionsOptions, dateTime, options) {
```

Comment:
is `dateTime` only available with `multimodal` costing option? then maybe it would be better to make it part of `costingoptions`?